src/daemon/ca/commands.rs

```diff
@@ -160,71 +176,93 @@ impl From<CmdDet> for StorableCaCommand {
             // ------------------------------------------------------------
             // Being a parent
             // ------------------------------------------------------------
-            CmdDet::ChildAdd(child, id_cert_opt, res) => {
-                StorableCaCommand::ChildAdd(child, id_cert_opt.map(|c| c.ski_hex()), res)
+            CmdDet::ChildAdd(child, id_cert_opt, resources) => StorableCaCommand::ChildAdd {
+                child,
+                ski: id_cert_opt.map(|c| c.ski_hex()),
+                resources,
+            },
+            CmdDet::ChildUpdateResources(child, resources) => {
+                StorableCaCommand::ChildUpdateResources { child, resources }
             }
-            CmdDet::ChildUpdateResources(child, res) => StorableCaCommand::ChildUpdateResources(child, res),
-            CmdDet::ChildUpdateId(child, id) => StorableCaCommand::ChildUpdateId(child, id.ski_hex()),
+            CmdDet::ChildUpdateId(child, id) => StorableCaCommand::ChildUpdateId {
+                child,
+                ski: id.ski_hex(),
+            },
             CmdDet::ChildCertify(child, req, _, _) => {
-                let (rcn, limit, csr) = req.unpack();
+                let (resource_class_name, limit, csr) = req.unpack();
                 let ki = csr.public_key().key_identifier();
```

Comment:
Given that you've expanded `rcn` to `resource_class_name` perhaps also expand the cryptic `ki` to `key_id`?